src/applications/check-in/hooks/useStorage.jsx

```diff
@@ -0,0 +1,181 @@
+import { useCallback, useMemo } from 'react';
+import { createStorageKeys } from '../utils/storage';
+
+const useStorage = (isPreCheckIn = true, local = false) => {
```

Comment:
Could we change this to take an object with named attributes for better readability when calling it?

Reply:
I thought about doing that. I had a conversation with someone (maybe Lee) during a different PR on what the threshold for having params as an object should be. I think we were talking something like 3 or 4. Since this was just 2, I decided against it. Happy to make the change but I think we should decide as a team what that should be and add that as a code style to our readme.